domains/_vercel.mack.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "mackcodes",
+        "email": "mackcodes"
```

```suggestion
        "email": "mackcodes"
```
This is not a valid email address.